### ISSUE TITLE
Cursor follow not supported

### ISSUE THREAD

**Issue by ElectroLund** [mark-wiemer/vscode-autohotkey-plus-plus#45]
Despite this option being checked, code follow doesn't work in the sidebar Outline.

![image](https://user-images.githubusercontent.com/2272478/107664526-bb8e2700-6c49-11eb-9d84-50bc45e46489.png)

For the record, this is not default behavior in other languages. In C, for instance, if you simply navigate in the editor into a new function block, the outline will dynamically update. This is faster as it requires less intentional double-clicking.

**Comment by Giessen** [comment]
This function still not working?

**Comment by mark-wiemer** [comment]
Correct, still not working. I've focused on other things during my maintenance of this project, but as long as the issue is open it's something I'm planning to add.
